src/ui/components/atoms/error-message/styles.scss

```diff
@@ -0,0 +1,131 @@
+@import '../../../../globals/variables';
+@import '../../../../globals/mixins';
+
+.error-message {
+  color: $ads-primary-main;
+  font-family: Arial, sans-serif;
+  padding: 10rem;
+  text-align: center;
+
+  h1 {
+    color: $ads-primary-main;
+    font-family: Inter, Arial, sans-serif;
```

Comment:
please review the figma I thing the font is Lato here

Reply:
![Captura de pantalla (740)](https://github.com/Ditmar/OpenScience/assets/130755635/1c4d11cd-ee9b-4d9b-ad54-728a1a25112e)
the font on the figma is Inter and not Lato